go/oasis-node/cmd/registry/entity/entity.go

```diff
@@ -42,6 +43,8 @@ var (
 	initFlags                 = flag.NewFlagSet("", flag.ContinueOnError)
 	updateFlags               = flag.NewFlagSet("", flag.ContinueOnError)
 	registerOrDeregisterFlags = flag.NewFlagSet("", flag.ContinueOnError)
+	submitFlags               = flag.NewFlagSet("", flag.ContinueOnError)
+	signedTxFlags             = flag.NewFlagSet("", flag.ContinueOnError)
```

Comment:
Consistency with staking CLI:
```suggestion
	txFileFlags             = flag.NewFlagSet("", flag.ContinueOnError)
```